frontend/src/ui/components/Header/Header.tsx

```diff
@@ -38,6 +40,12 @@ export const HeaderComponent: React.FC<Props> = (props: Props) => {
     );
   }
 
+  if (props.access !== '') {
+    userIsLogged = true;
+  } else {
+    userIsLogged = false;
+  }
+
```

Comment:
Do we really need an extra variable?

Reply:
Not really, I was failing in favor of being explicit, but thank you for suggesting removing this.